src/stream/src/executor/join/hash_join.rs

```diff
@@ -667,10 +668,28 @@ impl JoinEntryState {
         key: PkType,
         value: StateValueType,
     ) -> Result<&mut StateValueType, JoinEntryError> {
+        let mut removed = false;
+        if !crate::consistency::strict_consistency().is_on() {
```

Comment:
I guess there's no ambiguity for importing the function into the scope in order to make the code cleaner. 😕